Bundle/CoreBundle/Resources/doc/SETUP.md

```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
 ```
 
 - Enable the `localizeddate` filter:
+# Fresh install including base config and dependencies
+
+## symfony 2.8
+
+```sh
+mkdir -p ~/workspace/clientName/projectName && cd ~/workspace/clientName/projectName
+symfony new . 2.8
+git init
+git remote add origin ssh://git@gitlab.troopers.agency:22022/{ClientName}/{ProjectName}.git
+git add .
+git commit -m ":tada: fresh install"
+git push -u origin develop
+```
 
+## Victoire and its dependencies
+
+### Composer
+```
+composer require victoire/victoire \
```

Comment:
It's easier to copy and paste if it's on one line:

`composer require victoire/victoire friendsofsymfony/user-bundle:~2.0@dev sensio/distribution-bundle:^2.3 doctrine/orm --update-with-dependencies`